packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1119,7 +1125,7 @@ export default class BackupNg {
           parentId: taskId,
         },
         xapi.exportVm($cancelToken, snapshot, {
-          compress: job.compression === 'native',
+          compress: BACKUP_COMPRESSION_TO_XO_COMPRESS[job.compression],
```

Comment:
What if `job.compression` is `undefined`?

Reply:
compress will default to false.

Reply:
Not good enough, what if the default value is changed in `exportVm`?

`job.compression === undefined` does not mean default value, it means no compression.